include/stl2/detail/memory/construct_at.hpp

```diff
@@ -22,14 +22,14 @@ STL2_OPEN_NAMESPACE {
 	requires Constructible<T, Args...>
 	void __construct_at(T& t, Args&&... args) {
 		::new(const_cast<void*>(static_cast<const volatile void*>(detail::addressof(t))))
-			T(std::forward<Args>(args)...);
+			remove_reference_t<T>(std::forward<Args>(args)...);
```

Comment:
Not necessary. It looks like the `value_type_t`-based specification in the draft paper doesn't match the actual implementation.